src/Core/Transform/ScaleAnImage/Code.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+# =========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# =========================================================================*/
+
+import sys
+
+import itk
+
+if len(sys.argv) < 3:
+    raise Exception(f"Usage: {sys.argv[0]} inputFile outputFile")
+
+image = itk.imread(sys.argv[1], itk.UC)
+
```

Comment:
`Dimension = image.GetImageDimension()`, then use this in the `ScaleTransform` initialization to make the script dimension-agnostic.